dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -187,12 +190,20 @@ public void findOneRelsTest() throws Exception {
         getClient().perform(get("/api/core/communities/" + child1.getID().toString() + "/logo"))
                    .andExpect(status().isOk());
 
+        //Main community has no collections, therefore contentType is not set
         getClient().perform(get("/api/core/communities/" + parentCommunity.getID().toString() + "/collections"))
                    .andExpect(status().isOk());
 
         getClient().perform(get("/api/core/communities/" + child1.getID().toString() + "/collections"))
                    .andExpect(status().isOk())
                    .andExpect(content().contentType(contentType));
+
+        //child1 subcommunity has no subcommunities, therefore contentType is not set
+        getClient().perform(get("/api/core/communities/" + parentCommunity.getID().toString() + "/subcommunities"))
+                   .andExpect(status().isOk());
+
+        getClient().perform(get("/api/core/communities/" + child1.getID().toString() + "/subcommunities"))
+                   .andExpect(status().isOk());
```

Comment:
This should return a 204 but currently it returns a wrong 200, see also https://jira.duraspace.org/browse/DS-3808

I suggest to prepare the test for 204 and keep it commented with a note //FIXME enable when DS-3808 is solved